setup.py

```diff
@@ -123,28 +123,20 @@ def run(self):
 
 
 class build_ext(setuptools.command.build_ext.build_ext):
-    def build_extensions(self):
-        for i, ext in enumerate(self.extensions):
-            if ext.name == "nvfuser._C":
-                # NOTE: nvfuser pybind target is built with cmake, we remove the entry for ext_modules
-                del self.extensions[i]
-
-                # Copy nvfuser extension to proper file name
-                fullname = self.get_ext_fullname("nvfuser._C")
-                filename = self.get_ext_filename(fullname)
-                fileext = os.path.splitext(filename)[1]
-                cwd = os.path.dirname(os.path.abspath(__file__))
-                src = os.path.join(cwd, "nvfuser", "lib", "libnvfuser" + fileext)
-                dst = os.path.join(cwd, filename)
-                if os.path.exists(src):
-                    print(
-                        "handling nvfuser pybind API, copying from {} to {}".format(
-                            src, dst
-                        )
-                    )
-                    self.copy_file(src, dst)
-
-        setuptools.command.build_ext.build_ext.build_extensions(self)
+    def build_extension(self, ext):
```

Comment:
err. is this what's causing the issue `build_extentions` vs `build_extension`?

Reply:
or is it because I'm still calling `super().build_extensions(self)` and that I didn't clear `nvfuse._C` from extensions?

Reply:
https://github.com/NVIDIA/Fuser/wiki/Building-fuser-project#note-for-developers

Reply:
I do think it's possible to do the same by changing `build_extensions`. The very critical point is that whether or not to copy the artifact of `libnvfuser.so` (if on linux) to `self.build_lib`.